### PR TITLE
[Fix](merge-on-write) Fix some bugs about sequence column

### PR DESCRIPTION
**Pull request by bobhan1**
## Proposed changes

1. add checks and handling of sequence column in https://github.com/apache/doris/pull/21896 to insert statement in origin planner and Nereids planner.
2. disable drop sequence mapping column in schema change.
## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

**Review by zhannngchen — DISMISSED** [review]
LGTM

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by zhannngchen** [comment]
run p0

**Comment by dataroaring** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

File: regression-test/conf/regression-conf.groovy [hunk 1]
```diff
@@ -24,18 +24,17 @@ defaultDb = "regression_test"
 // init cmd like: select @@session.tx_read_only
 // at each time we connect.
 // add allowLoadLocalInfile so that the jdbc can execute mysql load data from client.
-jdbcUrl = "jdbc:mysql://127.0.0.1:9030/?useLocalSessionState=true&allowLoadLocalInfile=true"
-targetJdbcUrl = "jdbc:mysql://127.0.0.1:9030/?useLocalSessionState=true&allowLoadLocalInfile=true"
+jdbcUrl = "jdbc:mysql://127.0.0.1:9134/?useLocalSessionState=true&allowLoadLocalInfile=true"
```
Comment (zhannngchen) [inline regression-test/conf/regression-conf.groovy:27]: should not change this file

**Comment by dataroaring** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

**Review by zhannngchen — DISMISSED** [review]
LGTM

**Comment by bobhan1** [comment]
run buildall

**Review by zhannngchen — DISMISSED** [review]
LGTM

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run buildall

**Comment by bobhan1** [comment]
run p0

**Review by zhannngchen — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**